voice-client-javascript/assets/admin/actions.private.js

```diff
@@ -0,0 +1,84 @@
+const path = require("path");
+
+class Actions {
+  constructor(client, options) {
+    this.client = client;
+    this.options = options;
+  }
+
+  async initialize() {
+    let env = {};
+    console.log("Creating TwiML Application");
+    let results = await this.createTwimlApp(this.options);
+    env = Object.assign(env, results);
+    const voiceUrl = `https://${this.options.DOMAIN_NAME}${path.join(path.dirname(this.options.PATH, "client-voice-twiml-app"))}`;
```

Comment:
Won't `path.join` on Windows use `\` as the separator and cause trouble in a URL?